src/common/HaskellDo/Toolbar/View.hs

```diff
@@ -107,6 +110,38 @@ pathInput state = Ulmus.newWidget "pathInput" $ do
     projPath <- liftIO $ getValueFromId "#pathInput event input"
     return $ NewPath projPath
 
+fsTree :: State -> Widget Action
+fsTree state = Ulmus.newWidget "fsTree" $
```

Comment:
agreed, you think it's in the right place/under the correct name? `HaskellDo.Toolbar.FileSystemTree`